### DISCUSSION TITLE
File Not Found display for /meetings/ page on aastl.org

### DISCUSSION THREAD

**Discussion by L-squared** [code4recovery/12-step-meeting-list#1126] [Q&A] [answered]
HI all - Lois L. here asking for help for aastl.org who all of a sudden has file not found displaying on the page where the plugin meeting list normally displays - any ideas? We have latest version of the plugin 3.1414. No changes were made to the site beyond meeting updates. The site is using wp5.1.1

<img width="1621" alt="image" src="https://github.com/code4recovery/12-step-meeting-list/assets/10970074/94b53671-36d2-4de7-b52a-60e5136cc34a">

**Comment by joshreisner** [comment]
@L-squared try settings > permalinks > save

**Reply by L-squared** [reply]
@joshreisner in the TSML settings I don't see a permalink - I do see something in the import export that says where my meeting list is - says this: https://aastl.org/meetings/

**Reply by joshreisner** [reply, marked as answer]
@L-squared i mean in wordpress settings

**Reply by L-squared** [reply]
Thanks friend!!!

**Reply by joshreisner** [reply]
yay
